### PR TITLE
resolve duplicate schema issue and resolve some naming issues for `Azure.ResourceManager.Monitor`

### PR DESCRIPTION
**Pull request by ArcturusZhang**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md [hunk 1]
```diff
@@ -71,7 +77,10 @@ prepend-rp-prefix:
 - OperationType
 
 override-operation-name:
-  ActionGroups_GetTestNotifications: GetActionGroupTestNotifications
+  ActionGroups_GetTestNotifications: GetNotificationStatus
```
Comment (ArthurMa1978) [inline sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md:80]: What does `Test` mean here? No sure remove it is the right.

File: sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md [hunk 2]
```diff
@@ -193,4 +262,55 @@ directive:
...
+    where: $.definitions.DataCollectionEndpointResource.properties
+    transform: >
+      $["kind"] = undefined;
+  - from: dataCollectionRules_API.json
+    where: $.definitions.DataCollectionRuleResource.properties
+    transform: >
+      $["kind"] = undefined;
+```
+
+## Tag: package-monitor-track2
```
Comment (ArthurMa1978) [inline sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md:285]: Consider to use the Readme.md file in the swagger spec repo directly

Comment (ArcturusZhang) [reply sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md:292]: They do not have a tag with all latest swagger yet

File: sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md [hunk 3]
```diff
@@ -71,7 +78,10 @@ prepend-rp-prefix:
 - OperationType
 
 override-operation-name:
-  ActionGroups_GetTestNotifications: GetActionGroupTestNotifications
+  ActionGroups_GetTestNotifications: GetNotificationStatus
+  ActionGroups_PostTestNotifications: CreateNotifications
+  ActionGroups_CreateNotificationsAtResourceGroupLevel: CreateNotifications
+  ActionGroups_CreateNotificationsAtActionGroupResourceLevel: CreateNotifications
```
Comment (ArcturusZhang) [inline sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md:84]: @thomasp98296 could you please take a look at these changes?

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**